roles/openshift_health_checker/test/disk_availability_test.py

```diff
@@ -34,7 +46,7 @@ def test_cannot_determine_available_disk(ansible_mounts, extra_words):
     with pytest.raises(OpenShiftCheckException) as excinfo:
         DiskAvailability(fake_execute_module, task_vars).run()
 
-    for word in 'determine disk availability'.split() + extra_words:
+    for word in extra_words:
```

Comment:
Sure :)

It's a band-aid anyway, the real goal is to base on https://github.com/openshift/openshift-ansible/pull/4913 and have the tests look for "named" exceptions.